x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -644,7 +644,7 @@ describe('When on the Trusted Apps Page', () => {
           });
 
           it('should trigger the List to reload', async () => {
-            expect(coreStart.http.get.mock.calls[0][0]).toEqual(TRUSTED_APPS_LIST_API);
+            expect(coreStart.http.get.mock.calls[1][0]).toEqual(TRUSTED_APPS_LIST_API);
```

Comment:
since sendGetEndpointSecurityPackage now gets called, this is no longer the first